CHANGELOG-1.2.md

```diff
@@ -13,6 +13,7 @@ When cutting a new release of the parent `k8ssandra` chart update the `unrelease
 
 ## unreleased
 
+* [ENHANCEMENT] #654 Reduce initial delay of Stargate readiness probe
```

Comment:
nit: This should be moved down and grouped with the other `[ENHANCEMENT]` updates in the list.